modules/common/react/lib/Popper.tsx

```diff
@@ -60,10 +61,6 @@ export class Popper extends React.PureComponent<PopperProps> {
       return null;
     }
 
-    if (!this.props.portal) {
-      return this.renderPopper();
-    }
-
     // do not use defaultProps for containerElement because document may not be statically available
     // at require time in some testing environments; instead we safely default at runtime
     return ReactDOM.createPortal(this.renderPopper(), this.props.containerElement || document.body);
```

Comment:
Can we combine `render` and `renderPopper` now? The only reason they were separated was due to the portal conditional

Reply:
Nit: Can we change `document.body` to a default prop?